src/_includes/page-header.html

```diff
@@ -1,8 +1,8 @@
 <header id="page-header">
   {% include navigation-main.html %}
   {% if page.obsolete == true %}
-  <div class="alert alert-danger">
-    <h4 class="text-center">Content of this page might be obsolete.</h4>
+  <div class="alert alert-warning">
+    <h4 class="text-center">Some of the content of this page might be obsolete.</h4>
```

Comment:
Can we say "outdated" instead of "obsolete"?

Reply:
How about "... page might be outdated or obsolete."

Reply:
Some reformatting would be needed here, but we can do that later.
Kathy, consider adding a Material alert [icon](https://material.io/tools/icons/?style=baseline).

Reply:
_Outdated_ and _obsolete_ are synonyms; I'm fine with using _outdated_ or maybe _out of date_.